MantidPlot/src/TSVSerialiser.h

```diff
@@ -33,6 +33,17 @@
 
   File change history is stored at: <https://github.com/mantidproject/mantid>
 */
+
+struct CaseInsensitiveStringComparator {
+  bool operator()(const std::string &s1, const std::string &s2) const {
+#ifdef _MSC_VER
+    return stricmp(s1.c_str(), s2.c_str()) < 0;
+#else
+    return strcasecmp(s1.c_str(), s2.c_str()) < 0;
+#endif
```

Comment:
I've changed my mind on this one:
 1. Note that be above is not equality, but checking if s1 is less than s2. 
 2. Boost versions add locale support, which we are not bothered about and the C lib ones should therefore be more efficient in this case.
 3. Yes this isn't the nicest code, but after the next few commits there should be only one instance.

What do you think? If you don't agree I could find the equivalent compare function for boost?